eventuate-cdc-data-producer-wrappers/src/main/java/io/eventuate/cdc/producer/wrappers/kafka/TopicPartitionSender.java

```diff
@@ -0,0 +1,80 @@
+package io.eventuate.cdc.producer.wrappers.kafka;
+
+import io.eventuate.messaging.kafka.producer.EventuateKafkaProducer;
+
+import java.util.concurrent.*;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.BiConsumer;
+
+public class TopicPartitionSender {
+  private EventuateKafkaProducer eventuateKafkaProducer;
+  private ConcurrentLinkedQueue<TopicPartitionMessage> messages = new ConcurrentLinkedQueue<>();
+  private AtomicReference<TopicPartitionSenderState> state = new AtomicReference<>(TopicPartitionSenderState.IDLE);
+  private TopicPartitionMessage lastFailedMessage;
+
+  public TopicPartitionSender(EventuateKafkaProducer eventuateKafkaProducer) {
+    this.eventuateKafkaProducer = eventuateKafkaProducer;
+  }
+
+  public CompletableFuture<?> handleEvent(String topic, String key, String body) {
```

Comment:
Shouldn't `handleEvent` be called `sendMessage`?

Reply:
ok